source/extensions/filters/listener/proxy_protocol/proxy_protocol.h

```diff
@@ -71,6 +81,8 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   // The index in buf_ where the search for '\r\n' should continue from
   size_t search_index_{1};
 
+  int header_version_{-1};
```

Comment:
can we use an enum instead of sentinel values? or at least document the meaning of the sentinel values?